docs/dev/project_management.rst

```diff
@@ -15,6 +15,54 @@ track bugs, enhancements, support requests, and just about any other work that g
 into the project. Try to make sure that issues have informative tags so we can find
 them easily.
 
+
+-------------------------------------------------------------------------------
+Bug triage
+-------------------------------------------------------------------------------
+
+When a new issue is discovered, we need to determine how urgent it is to
+address.
+
+Our core offering is complete, connected, and granular data. Issues that
+interrupt the availability of that data are of highest importance.
+
+However, not all datasets in PUDL are the same; some are mature and have many
+downstream users; others are more experimental. We've split them into "tier 1"
+and "tier 2" groups below.
+
+**Tier 1 datasets**
+* FERC 1 schedules XYZ
```

Comment:
This is just some placeholder stuff. I think the main thrust of this proposal is:

1. We should decide which datasets are "important" and thus warrant firefighting action, vs. "unimportant" and thus get slotted in with the rest.
2. We should also decide what it means for something to be "broken" - i.e. X% data missing/incorrect, new data unincorporated after X time, etc. - **absolutely** I need help actually defining these
3. If we have 1. and 2. then it will be much easier for us to make prioritization decisions about random things that blow up!

Reply:
The issue here is that the most integrated data has the most testing infrastructure written into it and is the most likely to fail. I think more useful here would be that the first step of triage is actually to scope out the issue and write some proposed solutions. Then we make the step of deciding whether we want to fix it in the most minimal way (relax the restriction, xfail the test), actually fix the core issue, or implement a more extensive design change. I think the pause between "here's the problem and what needs to be done" and "which version of this fix should we implement now" is probably the thing we most often fail to do and would help us prioritize.